test/cookbooks/cerner_splunk_ingredient_test/recipes/splunk_app.rb

```diff
@@ -0,0 +1,27 @@
+splunk_app_custom 'test_app' do
+  configs(proc do
+    splunk_conf 'testing.conf' do
+      config(
+        debug: {
+          banana: 'yellow'
+        }
+      )
+    end
+  end)
+
+  files(proc do |app_path|
+    file Pathname.new(app_path).join('plain_file.txt').to_s do
```

Comment:
Can create a file like [nav.xml](https://github.com/cerner/cerner_splunk/blob/stable/vagrant_repo/data_bags/cerner_splunk/cluster-apps-vagrant.json#L66-L67), which resides few directories deep? I'm guessing we cannot because the parent directories (data/ui/views/) are not being created here.

Also if I'm understanding this, _files_ property can be used only to create files directly under the app directory. Are there files that we create under the app directory? Shouldn't they all be folders under the app directory (I might be wrong though)?